roles/openshift_master/tasks/main.yml

```diff
@@ -114,12 +121,26 @@
 
 - name: Start and enable openshift-master
   service: name=openshift-master enabled=yes state=started
+  when: not openshift_master_ha
   register: start_result
 
 - name: pause to prevent service restart from interfering with bootstrapping
   pause: seconds=30
   when: start_result | changed
 
+- name: Install cluster packagese
+  yum: pkg=pcs state=present
+  when: openshift_master_ha and not openshift.master.cluster_defer_ha | bool
+  register: install_result
+
+- name: Start and enable cluster service
+  service: name=pcsd enabled=yes state=started
+  when: openshift_master_ha and not openshift.master.cluster_defer_ha | bool
+
+- name: Set the cluster user password
+  shell: echo {{ openshift_master_cluster_password | quote }} | passwd --stdin hacluster
```

Comment:
Acknowledging that this is a shell call, but that you're using the quote filter so we should be ok.

Longer term, I'd like to find a way to get this done with the [ansible user module](http://docs.ansible.com/user_module.html), but that requires an already encrypted PW, and I wasn't able to find (in my 5 minute search) a viable one to use here.